frontend/awx/views/schedules/types.ts

```diff
@@ -38,13 +38,14 @@ export interface RuleFields {
   bysetpos: null;
   byminute: null;
   byhour: null;
-  until: { date: string; time: string } | null;
-  count: null;
   rules: RuleListItemType[] | [];
   exceptions: RuleListItemType[] | [];
 }
 export interface ScheduleFormWizard {
   resourceInventory?: number;
+  endType: string | undefined;
+  until: { date: string; time: string } | null;
+  count: number | null;
```

Comment:
This will need to be move [here](https://github.com/ansible/ansible-ui/pull/2845/files#diff-e3366e082af0834bc3bded721ac34363eb02dd0c2740f5fb971f84c0e5e23822R28)
```suggestion
```